### ISSUE TITLE
The cursor is not visible when it's at the beginning of a line.

### ISSUE THREAD

**Issue by m5x5** [dokieli/dokieli#586]
In the video you see a demo of the issue.
My first guess would be to use outline-offset: 4px. Or to hide the focus alltogether if this will not negatively impact accessibility.

https://github.com/user-attachments/assets/5237263a-9984-4557-9e02-b6a7a06f49d5

**Comment by csarven** [comment]
Excellent. Thanks for raising this. Fixed.
